README.md

```diff
@@ -269,6 +272,9 @@ class TritonPythonModel:
                 auto_complete_model_config.add_output(output)
 
         auto_complete_model_config.set_max_batch_size(0)
+        
+        # Uncomment this line when setting max_batch_size > 1
```

Comment:
"To enable a dynamic batcher with default settings, you can use `auto_complete_model_config.set_dynamic_batching()` function. It is commented in this example because the max_batch_size is zero"

Reply:
Updated.

Reply:
Thanks Kyle!